src/components/UI/NetworkBar/NetworkBar.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from "react";
+import styled from "@emotion/styled";
+import { vars } from "../../../styles";
+import { SvgIcon, SvgIconX } from "../SvgIcon";
+
+const NetworkWrap = styled.div`
+  background-color: #001f29;
+  color: ${vars.white};
+  padding: 8px 0;
+
+  .network {
+    text-transform: capitalize;
+  }
+
+  svg {
+    cursor: pointer;
+
+    &:hover {
+      color: ${vars.grey};
+    }
+  }
+`;
+
+interface NetworkBarProps {
+  network?: string;
+}
+
+export const NetworkBar: React.FunctionComponent<NetworkBarProps> = ({
+  network,
+}: NetworkBarProps) => {
+  const [show, setShow] = useState(false);
+
+  useEffect(() => {
+    if (network) {
+      setShow(true);
+    } else {
+      setShow(false);
+    }
+  }, [network]);
```

Comment:
I think dont need this use effect leh, cos hor the network wont change and nothing will set it back right?
I think change to something like this can liao, cos like overkill using useEffect().
```
if (network) setShow(true);
```
U try it out, should have the same effect.

Reply:
was thinking about the scenario when user click x, then use another config file then it wont show up again le  🤔

Reply:
hmm, ok la, but hor if change config file liao it is still showing up. im ok with it the current implementation though.